wandb/sdk/internal/sender.py

```diff
@@ -1209,6 +1216,15 @@ def send_link_artifact(self, record: "Record") -> None:
             except Exception as e:
                 logger.warning("Failed to link artifact to portfolio: %s", e)
 
+    def send_use_artifact(self, record: "Record") -> None:
+        """
+        This function doesn't actually send anything, it is just used
+        internally
+        """
+        use = record.use_artifact
+        if use.type == "job":
+            self._job_builder.used_job = True
```

Comment:
any idea why this one is not covered?

Reply:
I added a test to specifically hit this code path

Reply:
yeah... if you feel confident this is one is covered is all good, just seemed weird that this one is not being hit